README.md

```diff
@@ -30,6 +26,8 @@
 
 [SonarQube-Scanner-Swift Code Coverage](swift-coverage/)
 
+**_NOTE:_** All examples related to C, C++ and Objective-C can be found here: [sonarsource-cfamily-examples](https://github.com/sonarsource-cfamily-examples)
```

Comment:
```suggestion
**_NOTE:_** All SonarQube Scanner examples for C, C++ and Objective-C can be found [here](https://github.com/sonarsource-cfamily-examples?q=-sq).
```

Reply:
The scope of this repo seems to be primarily SQ scanner examples. I restricted the link to those. Users can find out by reading the README there that we cover a wider scope.
WDYT?

Reply:
I hesitated TBH :)
So that's fine with me!